pkg/apis/allocation/v1/zz_generated.deepcopy.go

```diff
@@ -18,11 +18,11 @@
 
 // Code generated by deepcopy-gen. DO NOT EDIT.
 
-package v1alpha1
+package v1
 
 import (
-	stable_v1alpha1 "agones.dev/agones/pkg/apis/stable/v1alpha1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1alpha1 "agones.dev/agones/pkg/apis/stable/v1alpha1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
we've used `metav1` everywhere else (pretty sure that's what k8s uses too as an alias)
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Reply:
oh wait, this is generated. Ignore this.

Reply:
I'm not sure why the generated code decided to change this import name, but I'd rather not fight with it. :)